data/ka'het missions.txt

```diff
@@ -0,0 +1,220 @@
+		sprite label/graveyard
+
+mission "Disabled Ka'het"
+	boarding
+	invisible
+	source
+		government "Ka'het" " Ka'het "
+	destination "Earth"
+	on offer
+		log "Successfully boarded a Ka'het, it appears to be possible by killing the organism without blowing up its artificial exoskeleton."
+		conversation
+			`Your ship slowly approaches the <origin>, now floating lifeless near you. As you move closer, you start noticing many holes in the weakest points of the hull and damages reaching what appears to be a large, slug-like being in the middle of the ship. While most of the ship's machinery doesn't seem to have received heavy damages, your weapons must have hit the slug's vital organs, killing it.`
+			`	You have no troubles docking with what seems to be its "exoskeleton", trying to decide what to do next: while the Ka'het is too different from a ship for you to command it, it's safe to say that you can now loot its outfits.`
+				decline
+
+mission "First Contact: Ka'het: Remnant 1A"
+	landing
+	minor
+	invisible
+	source
+		government "Remnant"
+	to offer
+		has "Graveyard Label: offered"
+		"remnant met taely" == 0
+		not "Remnant: Technology Available: offered"
+	on offer
+		conversation
+			`Do you want to ask the Remnant about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You ask around more than a dozen people, but none of them seems to be aware of what you're talking about. Either the information is too confidential to be given to a stranger like you, or you didn't ask the right people.`
+				decline
+
+mission "First Contact: Ka'het: Remnant 1B"
+	landing
+	minor
+	invisible
+	source "Viminal"
+	to offer
+		has "Graveyard Label: offered"
+		"remnant met taely" >= 1
+	on offer
+		conversation
+			`Do you want to ask Taely about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You find Taely inside one of the outern hangars of the shipyard, apparently teaching a small group of people how to operate the basic repairs over an Albatross. You sit on a bench waiting for her to finish her lesson, and in a few minutes you see her walking towards you. "Hello, <first>!" she says. "Sorry, but I have nothing for you to do today." "Actually," you reply, "I'm here because I need to make you a few questions."`
+			`	As she sits near you, you start explaining her about the area you reached, and the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existance of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`
```

Comment:
```suggestion
			`	As she sits near you, you start explaining her about the area you reached, and the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existence of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands of years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`
```

> but with some tech so advanced to still be conserved enough to be reverse engineered.

This is a perfect example of what I am intending when I say that the Builders could have left behind massive pristine cities. If the Ka'het technology is able to survive this long in such a good condition, why not their cities?